app/_hub/kong-inc/vault-auth/index.md

```diff
@@ -33,37 +33,44 @@ params:
     - name: access_token_name
       required: true
       default: "`access_token`"
+      datatype: array of string elements
       description: |
         Describes an array of comma-separated parameter names where the plugin will look for an access token. The client must send the access token in one of those key names, and the plugin will try to read the credential from a header or the querystring parameter with the same name.<br>*note*: the key names may only contain [a-z], [A-Z], [0-9], [_] and [-].
     - name: vault.id
       required: true
       default:
       value_in_examples: "<UUID>"
+      datatype: foreign
       description: |
         A reference to an existing `vault` object within the database. `vault` entities define the connection and authentication parameters used to connect to a Vault HTTP(S) API.
     - name: secret_token_name
       required: true
       default: "`secret_token`"
+      datatype: array of string elements
       description: |
         Describes an array of comma-separated parameter names where the plugin will look for a secret token. The client must send the secret in one of those key names, and the plugin will try to read the credential from a header or the querystring parameter with the same name.<br>*note*: the key names may only contain [a-z], [A-Z], [0-9], [_] and [-].
     - name: key_in_body
```

Comment:
I don't see this field in the plugin's schema 🤔 

There is `tokens_in_body` though. Perhaps a typo from the past?

Reply:
> I don't see this field in the plugin's schema 🤔
> 
> There is `tokens_in_body` though. Perhaps a typo from the past?

Great catch!

Reply:
I should've noticed that myself. Just testing you! Just kidding! Done in commit 6372df2.